iothub/device/src/Transport/Http/HttpClientHelper.cs

```diff
@@ -59,6 +75,9 @@ internal sealed class HttpClientHelper : IHttpClientHelper
             _authenticationHeaderProvider = authenticationHeaderProvider;
             _defaultErrorMapping = new ReadOnlyDictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>>(defaultErrorMapping);
 
+            ServicePoint servicePoint = ServicePointManager.FindServicePoint(_baseAddress);
```

Comment:
Is this also a work around a user could make in their own app if they are using an SDK that doesn't do this?